packages/testring-recorder-backend/src/recorder-server.ts

```diff
@@ -141,6 +172,17 @@ export class RecorderServer implements IRecorderServer {
     }
 
     public openBrowser(): void {
-        opn(this.httpServer.getUrl());
+        const browserDir = path.resolve(__dirname, `../temp/chromeUser/rcExt${nanoid()}`);
```

Comment:
Please, fix opening for macOS and Windows

Reply:
Done. Please try on different OSes